src/db.c

```diff
@@ -1233,6 +1231,46 @@ void scanGenericCommand(client *c, robj *o, unsigned long long cursor) {
             p = lpNext(o->ptr, p);
         }
         cursor = 0;
+    } else if (o->type == OBJ_HASH &&
+                (o->encoding == OBJ_ENCODING_LISTPACK ||
+                 o->encoding == OBJ_ENCODING_LISTPACK_TTL)) {
+        int expired;
+        int64_t len;
+        long long expireAt;
```

Comment:
we use snake style in redis.
```suggestion
        long long expire_at;
```

Reply:
I copied it but it makes me doubt as well. I see there are bunch of mixed style examples in the unstable branch as well but if the most common is snake case, let's change our code for it. @moticless please ack so we can have consistent style for hfe code.

Reply:
I am fine (I try to align to the file I am within. If it is a new file, I prefer camel-case.)